env.template

```diff
@@ -1,3 +1,5 @@
+ALERTS=localhost
+ALL_STOPS=localhost
```

Comment:
we should just call this `TCAT_SERVICE` instead of having one per alerts, all_stops, and live_tracking